pandas/tests/io/parser/python_parser_only.py

```diff
@@ -19,6 +19,15 @@
 
 class PythonParserTests(object):
 
+    def test_default_separator(self):
+        # GH17333
```

Comment:
@Licht-T : Actually, before we merge this, can you add your explanation for why this behavior is correct?  For future developers, we should ensure that they understand.